src/index.js

```diff
@@ -1,12 +1,23 @@
 const { default: axios } = require('axios');
 const express = require('express');
 const cors = require('cors');
+const auth = require('express-basic-auth');
 require('dotenv').config();
 
 const app = express();
 const API_URL = process.env.API_URL || 'https://trainees-monitor-dev.herokuapp.com/register';
 const port = process.env.PORT || 3000;
 const origin = process.env.ORIGIN;
+const username = process.env.BASIC_AUTH_USERNAME;
+const password = process.env.BASIC_AUTH_PASSWORD;
+
+app.use(auth({
+  users: { [username]: password },
+  challenge: true,
+}));
+
+console.log(username);
+console.log(password);
```

Comment:
Do we need these logs now?

**Side note:** it's dangerous to log sensitive data, as logs sometimes can be accessed by other developers (for debugging purposes) who shouldn't see these secrets.